corehq/apps/hqwebapp/templates/hqwebapp/includes/domain_list_dropdown.html

```diff
@@ -10,7 +10,7 @@
     {% endif %}
     {% endif %}
 
-    {% if not enterprise_mode %}
+    {% if not enterprise_mode and not restrict_domain_creation %}
```

Comment:
Oh this is weird - so this is already hidden on enterprise mode?  Oddly, I still see that link.  I think it should be `or not restrict_domain_creation`, since I would expect `restrict_domain_creation` to always do what it says, regardless of the status of `enterprise_mode` (although I can't think of an instance where you'd the latter set and not the former).

Reply:
Oops, updated